content/en/llm_observability/evaluations/managed_evaluations/_index.md

```diff
@@ -93,6 +93,23 @@ Connect your Amazon Bedrock account to LLM Observability with your AWS Account.
 
 {{< img src="llm_observability/configuration/amazon-bedrock-tile.png" alt="The Amazon Bedrock configuration tile in LLM Observability. Lists instructions for configuring Amazon Bedrock." style="width:100%;" >}}
 
+[1]: https://app.datadoghq.com/llm/settings/integrations
+{{% /tab %}}
+{{% tab "AI Gateway" %}}
+<div class="alert alert-info">If you are subject to HIPAA, you are responsible for ensuring that you connect only to an AI Gateway that is subject to a business associate agreement (BAA) and meets all requirements for HIPAA compliance.</div>
```

Comment:
I feel like the note could elevated to a "Caution": https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown+Shortcode+Reference#Alert-boxes

```suggestion
<div class="alert alert-danger">If you are subject to HIPAA, you are responsible for ensuring that you only connect to an AI Gateway that is subject to a business associate agreement (BAA) and meets all requirements for HIPAA compliance.</div>
```